tests/x11/thunderbird.pm

```diff
@@ -16,8 +16,10 @@ sub run() {
     ensure_installed("MozillaThunderbird");
     x11_start_program("thunderbird");
     assert_screen 'test-thunderbird-1', 3;
-    send_key "alt-f4",                  1;    # close wizzard
-    send_key "alt-f4",                  1;    # close prog
+    wait_screen_change {
+        send_key "alt-f4";    # close wizzard
+    }
+    send_key "alt-f4";        # close prog
```

Comment:
* shouldn't both be guarded with wait_screen_change?
* Didn't you tell me that "wait_screen_change {}" needs to be closed with ';'?